protoc-gen-grpc-gateway/gengateway/template.go

```diff
@@ -248,6 +248,9 @@ var (
 	handleSend := func() error {
 		var protoReq {{.Method.RequestType.GoType .Method.Service.File.GoPkg.Path}}
 		err = dec.Decode(&protoReq)
+		if err == io.EOF {
```

Comment:
Is it better to handle `io.EOF` at L264 `if err := handleSend(); err != nil ` and return `nil` instead of `io.EOF`?